Makefile

```diff
@@ -420,7 +430,7 @@ $(addprefix docker-build-, $(IMAGES)): docker-build-%: %
 docker-build-ui:
 	mkdir -p build; test -e dashboard || (git clone $(UI_REPO) build/ffdl-ui; ln -s build/ffdl-ui dashboard)
 	(cd dashboard && (if [ "$(VM_TYPE)" = "minikube" ]; then eval $$(minikube docker-env); fi; \
-		docker build -q -t $(DOCKER_REPO)/$(DOCKER_NAMESPACE)/$(IMAGE_NAME_PREFIX)ui:$(IMAGE_TAG) .; \
+		docker build -t $(DOCKER_REPO)/$(DOCKER_NAMESPACE)/$(IMAGE_NAME_PREFIX)ui:$(IMAGE_TAG) .; \
```

Comment:
@sboagibm This is the place that causes the `docker-build-ui` to have segmentation fault on Travis. I'm not sure is it because Travis has bugs on Docker logging or lack of console/shell memory, but quiet the docker build standard output should let you pass the Travis build. (Tested on my own fork)